### PR TITLE
[FreeRDP] update to 2.3.2

### PR DESCRIPTION
**Pull request by zhongjohnson**
**Describe the pull request**

Update FreeRDP to the latest version 2.3.2

Note : no feature need to test

### CODE REVIEW

File: ports/freerdp/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "freerdp",
-  "version-semver": "2.2.0",
+  "version-semver": "2.3.2",
   "port-version": 1,
```
Comment (PhoebeHui) [inline ports/freerdp/vcpkg.json:4]:
```suggestion
  "port-version": 0,
```

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
@zhongjohnson, thanks for your PR!

Could you update the port version? please submit the changes, then execute './vcpkg x-add-version freerdp' to update the baseline version and submit the changes again.

File: ports/freerdp/vcpkg.json (hunk 1 above)
Comment (zhongjohnson) [reply ports/freerdp/vcpkg.json]: Fixed, thanks for your comments.

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
The hash looks wrong, please execute './vcpkg x-add-version --overwrite-version freerdp', and submit the changes again.

File: versions/f-/freerdp.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "0faaf0587e1e9389b7fded477be9ba088ce76b5e",
```
[the remark this answers is not in the document]
Comment (zhongjohnson) [reply versions/f-/freerdp.json]: Fixed.

**Comment by zhongjohnson** [comment]
> @zhongjohnson, could you check the cla? may need resign again?

I didn't find where to "resign" the CLA, let's just cancel this PR.

**Comment by PhoebeHui** [comment]
Related to https://github.com/microsoft/vcpkg/pull/18242

**Pull request closed without merging**